nncf/quantization/advanced_parameters.py

```diff
@@ -128,6 +128,8 @@ class AdvancedQuantizationParameters:
     :type inplace_statistics: bool
     :param disable_bias_correction: Whether to disable the bias correction.
     :type disable_bias_correction: bool
+    :param alpha: SmoothQuantize-related parameter.
```

Comment:
Please add relevant description that user can understand meaning of the parameter and that the parameter must be changed if the obtained quantized model is not accurate. The description will be added to the developer documentation.